data/assets/scene/solarsystem/heliosphere/bastille_day/fieldlines.asset

```diff
@@ -0,0 +1,83 @@
+local assetHelper = asset.require('util/asset_helper')
+local heliosphereTransforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+
+
+local transferFunctions = asset.localResource("transferfunctions")
+local masVelocityColorTable = transferFunctions .. "/velocity_fieldlines.txt"
+local masDensityColorTable = transferFunctions .. "/density_fieldlines.txt"
+
+local fieldlinesDirectory = asset.syncedResource({
+    Name = "Bastille Day MAS Fieldlines",
+    Type = "HttpSynchronization",
+    Identifier = "bastille_day_mas_fieldlines",
+    Version = 2
+})
+
+local sunRadius = 6.957E8
+
+-- Fieldlies from binaries
+local fieldlines = {
+    Identifier = "MAS_MHD_Fieldlines",
+    Parent = heliosphereTransforms.HNMReferenceFrame.Identifier,
+    --wrong Parent = sunTransforms.SunIAU.Identifier,
+    Renderable = {
+        Type = "RenderableFieldlinesSequence",
+        SourceFolder = fieldlinesDirectory,
+        -- FlowEnabled = false,
+        -- LineWidth = 1.0;
+        AlphaBlendlingEnabled = false,
+        InputFileType = "osfls",
+        ColorTablePaths = {
+            masDensityColorTable,
+            masVelocityColorTable,
+        },
+        ColorTableMinMax = {
+            { 0, 1000000 },
+            { 100, 2000 }
+        },
+        LoadAtRuntime = true,
+        ScaleToMeters = 1.0,
+        SecondsBefore = 24*60*60,
+        SecondsAfter = 24*60*60,
+        SimulationModel = "mas",
+        Color = {
+            Uniform = {0, 0.725, 0.75, 1}
+        }
+    },
+    GUI = {
+        Path = "/Solar System/Heliosphere",
+        Name = "MAS Fieldlines"
+    },
+    Transform = {
+        Scale = {
+            Type = "StaticScale",
+            Scale = sunRadius
+        },
+	    Rotation = {
```

Comment:
Yes, that's a tab instead of space. Github has the option to show whitespaces (really useful) and then it looks like this: 
![image](https://user-images.githubusercontent.com/8808894/124893365-f454e100-dfda-11eb-8d8f-73288da42388.png)